supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala

```diff
@@ -146,21 +150,25 @@ class TesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
   }
 
   override def pollStatusAsync(handle: StandardAsyncPendingExecutionHandle)(implicit ec: ExecutionContext): Future[TesRunStatus] = {
-    val pollTask = pipeline[TesGetResponse].apply(Get(s"$tesEndpoint/${handle.pendingJob.jobId}"))
+    val pollTask = pipeline[Task].apply(Get(s"$tesEndpoint/${handle.pendingJob.jobId}"))
 
     pollTask.map {
       response =>
         val state = response.state
         state match {
-          case s if s.contains("Complete") =>
+          case s if s.contains("COMPLETE") =>
             jobLogger.info(s"Job ${handle.pendingJob.jobId} is complete")
             Complete
 
-          case s if s.contains("Cancel") =>
+          case s if s.contains("CANCELED") =>
             jobLogger.info(s"Job ${handle.pendingJob.jobId} was canceled")
             FailedOrError
 
-          case s if s.contains("Error") =>
+          case s if s.contains("ERROR") =>
+            jobLogger.info(s"TES reported an error for Job ${handle.pendingJob.jobId}")
+            FailedOrError
+
+          case s if s.contains("SYSTEM_ERROR") =>
```

Comment:
This case will never hit because `"ERROR"` (checked above) is a substring of `"SYSTEM_ERROR"`.

Since they're the same handler anyway, I'd just remove this case.